requirements/download-deps/bootstrap-win-lock.txt

```diff
@@ -1,24 +1,24 @@
 #
-# This file is autogenerated by pip-compile with Python 3.13
+# This file is autogenerated by pip-compile with Python 3.9
 # by the following command:
 #
-#    pip-compile --allow-unsafe --config=pyproject.toml --generate-hashes --output-file='requirements\download-deps\bootstrap-win-lock.txt' --unsafe-package=flit-core --unsafe-package=pip --unsafe-package=setuptools --unsafe-package=wheel 'requirements\download-deps\bootstrap.txt'
+#    pip-compile --allow-unsafe --generate-hashes --output-file=requirements/download-deps/bootstrap-win-lock.txt --unsafe-package=flit-core --unsafe-package=pip --unsafe-package=setuptools --unsafe-package=wheel requirements/download-deps/bootstrap.txt
 #
 
 # The following packages are considered to be unsafe in a requirements file:
 flit-core==3.9.0 \
     --hash=sha256:72ad266176c4a3fcfab5f2930d76896059851240570ce9a98733b658cb786eba \
     --hash=sha256:7aada352fb0c7f5538c4fafeddf314d3a6a92ee8e2b1de70482329e42de70301
-    # via -r requirements\bootstrap.txt
-pip==24.3.1 \
-    --hash=sha256:3790624780082365f47549d032f3770eeb2b1e8bd1f7b2e02dace1afa361b4ed \
-    --hash=sha256:ebcb60557f2aefabc2e0f918751cd24ea0d56d8ec5445fe1807f1d2109660b99
-    # via -r requirements\bootstrap.txt
+    # via -r requirements/download-deps/../bootstrap.txt
+pip==25.1.1 \
+    --hash=sha256:2913a38a2abf4ea6b64ab507bd9e967f3b53dc1ede74b01b0931e1ce548751af \
+    --hash=sha256:3de45d411d308d5054c2168185d8da7f9a2cd753dbac8acbfa88a8909ecd9077
+    # via -r requirements/download-deps/../bootstrap.txt
 setuptools==71.1.0 \
     --hash=sha256:032d42ee9fb536e33087fb66cac5f840eb9391ed05637b3f2a76a7c8fb477936 \
     --hash=sha256:33874fdc59b3188304b2e7c80d9029097ea31627180896fb549c578ceb8a0855
-    # via -r requirements\download-deps\bootstrap.txt
+    # via -r requirements/download-deps/bootstrap.txt
```

Comment:
This is usually a bad sign, it looks like we generated this on a *nix-style system rather than Windows. I don't know if it materially changes what the lockfile contains, but we'll want to make sure we make any updates from the right operating system.